src/content/reference/docker-compose.mdx

```diff
@@ -387,6 +387,8 @@ volumes:
 Host volumes are added to your containers at deployment time.
 In addition, `okteto up` uses host volumes to infer the [sync paths](reference/okteto-manifest.mdx#sync-string-required) between your local filesystem and your remote development container.
 
+By default, services mounting the same volume will be placed on the same node using Kubernetes's pod affinity. If you wish to disable this behaviour, set `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED` to `false`.
```

Comment:
@andreafalzetti could we specify where this should be added? Is this on the command, on the compose? 

Could this be added as an Okteto secret or catalog configuration?

Reply:
Resolved with @omnomagonz in 48629cb48d9b100f27ed008cbd5709dffb51bd3e